PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java

```diff
@@ -112,21 +113,39 @@ public void onCreate(Bundle savedInstanceState) {
                 .setOnItemClickListener(new AdapterView.OnItemClickListener() {
                     @Override
                     public void onItemClick(AdapterView<?> arg0, View view,
-                                            int position, long id) {
-                        if (answers.get(position).getNextQuestionID() > 0) {
-                            // Next Question
-                            SessionHistory.currQID = answers.get(position)
-                                    .getNextQuestionID();
-                            updatePoints(position);
-                            updateQA();
-
+                                            final int position, long id) {
+                        if (answers.get(position).getPoints() == 0 &&
+                            !(answers.get(position).getAnswerDescription().equals(R.string.returns_home))) {
```

Comment:
try matches instead of equals since on running your code it's showing a snackbar even on selection "Returns Home"